src/enzyme_ad/jax/Implementations/HLODerivatives.td

```diff
@@ -705,7 +705,7 @@ def : HLODerivative<"SqrtOp", (Op $x),
 
 def : HLODerivative<"TanhOp", (Op $x),
                     [
```

Comment:
Is there a reason for this. Mul seems like it would be more efficient?

Reply:
Nah, I just tried to be more consistent mathematically. I can turn it back to `Mul` but shouldn't some pass change the square to a `Mul` already?

Reply:
Not presently so it’s worth keeping it like this for now

Reply:
Also it may hit an issue in complex since the constant needs to be the correct type